resource-managers/mesos/src/main/scala/org/apache/spark/scheduler/cluster/mesos/MesosClusterScheduler.scala

```diff
@@ -430,6 +429,7 @@ private[spark] class MesosClusterScheduler(
   }
 
   private def getDriverUris(desc: MesosDriverDescription): List[CommandInfo.URI] = {
+    val useFetchCache = desc.conf.get(config.ENABLE_FETCHER_CACHE)
```

Comment:
This is completely wrong. The following is the Apache Spark patch.
```
    val useFetchCache = desc.conf.get(config.ENABLE_FETCHER_CACHE) ||
	        conf.get(config.ENABLE_FETCHER_CACHE)
```

Reply:
My mistake - I missed a couple commits in my cherry-pick. Will fix.

Reply:
Thanks